partiql-lang-kotlin-omitted-tests.md

```diff
@@ -0,0 +1,70 @@
+The following lists out the tests from `partiql-lang-kotlin`'s parse tests that are not part of `partiql-tests`:
+
+Tests excluded:
+- Custom type cast
+  - https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L631-L644
+  - [SqlParserCastTests.kt](https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserCastTests.kt)
+  - [SqlParserCustomTypeCatalogTests.kt](https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserCustomTypeCatalogTests.kt) 
+- ORDER BY tests testing defaults for sort spec and null spec
+  - https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L1793-L1888
+  - dependent on when parsed ast checking is added
+- Many nested NOT regression test
+  - https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L4239-L4271
+  - performance test shouldn't be included in parse tests
+- `expectedSelectMissingFrom` -- not part of the PartiQL spec
+https://github.com/partiql/partiql-lang-kotlin/blob/34625c68dbcbaf7b8ae60df7a4cf65c60b2a3b79/lang/test/org/partiql/lang/errors/ParserErrorsTest.kt#L403-L415
```

Comment:
This is just `SELECT a FROM data` which has had a 'typo' dropping the `FROM`, right?